src/src/DataStructs/SettingsStruct.cpp

```diff
@@ -5,6 +5,7 @@
 #include "../CustomBuild/ESPEasyLimits.h"
 #include "../DataStructs/DeviceStruct.h"
 #include "../../ESPEasy_common.h"
+#include "../WebServer/HardwarePage.h"
```

Comment:
Hmm this is strange, why should the SettingsStruct import something from `WebServer` ?
If that's really needed, then we must move stuff as it makes no sense to have some code related to viewing things included in a settings struct.
It should be the other way around.

Reply:
Maybe the enum should be defined in another .h file, this seemed the 'closest' to where it is used most, but I may have picked the wrong file.